libs/application/templates/rental-agreement/src/lib/dataSchema.ts

```
@@ -0,0 +1,25 @@
+import * as m from './messages'
+
+export const dataSchema = z.object({
+  approveExternalData: z.boolean().refine((v) => v),
+  person: z.object({
+    nationalId: z.string().refine((n) => n && !kennitala.isValid(n), {
+      params: m.dataSchema.nationalId,
+    }),
+    name: z.string().min(2).max(256),
+    age: z.string().refine((x) => {
+      const asNumber = parseInt(x)
+      if (isNaN(asNumber)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)